build/run.js

```diff
@@ -296,6 +316,11 @@ optParser.options('dev', {
     type     : 'bool',
 })
 
+optParser.options('target', {
+    describe: 'Set the build target. Defaults to current platform',
```

Comment:
not a sentence.

Reply:
Plus, imo its worth to enuerate what platforms are supported, because platform names are not obvious